t/02_test_tests.t

```diff
@@ -0,0 +1,85 @@
+use strict;
+use warnings;
+use Test::Builder::Tester tests => 10;
+use Test::Most;
+use Email::Stuffer;
+use Email::Stuffer::TestLinks;
+use Email::Sender::Transport::Test;
+
+my $transport = Email::Sender::Transport::Test->new;
+
+sub make_msg {
+    my @links = @_;
+    return @_
+      ? join "\n", map { "This is a <a href=\"$_\">link</a>." } @links
+      : 'There are no links.';
+}
+
+sub make_teststr {
+    my @links = @_;
+    my $c     = 1;
+    return join "\n",
+      map { "ok " . $c++ . " - Link in email works ($_)" } sort @links;
+}
+
+sub send_it {
+    my $msg = shift;
+    Email::Stuffer->from('sender@example.com')->to('recipient@example.org')
+      ->html_body($msg)->transport($transport)->send_or_die;
+}
+
+my @urls = ();
+
+test_out("ok 1 - it died");
```

Comment:
This is an odd way of writing tests - why not just use the regular `Test::More` functionality?

Reply:
Done, although the "tests of tests" still have to use this way